src/plugins/maps_legacy/server/index.ts

```diff
@@ -57,6 +57,7 @@ export const config: PluginConfigDescriptor<MapsLegacyConfig> = {
       'map.regionmap.includeElasticMapsService',
       'map.regionmap.includeOpenSearchMapsService'
     ),
+    renameFromRoot('showRegionBlockedWarning', 'showRegionDeniedWarning'),
```

Comment:
i'm fine with this, I'm pretty sure only OpenSearch Dashboards core uses this configuration.

Reply:
Actually I'm not currently to pull the config, there are still code changes that we need to update but even when I try locally I can't get it. I've opened an [issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2039), I'm not positive but it seems like we cannot get in maps

Reply:
so we are not changing anything here?

Reply:
i've updated main with the fix so we should be good to go for verification purposes. but this will still need to be updated to `renameFromRoot('map.showRegionBlockedWarning', 'map.showRegionDeniedWarning')` for example try setting `map.showRegionDeniedWarning: true` in `opensearch_dashboards.yml` this will fail to start.